package Message; 
message ErrorNo 								  		//错误信息
{ 
   optional int32			error				= 1;	//错误码
   optional	int64			userid			= 2;	//账号ID
   optional int32			type				= 3;	//信息类型
   optional int32			paramint1		= 4;	//扩展参数
   optional int32			paramint2		= 5;	//扩展参数
   optional int32			paramint3		= 6;	//扩展参数
   optional int64			paramlong1	= 7;	//扩展参数
   optional int64			paramlong2	= 8;	//扩展参数
   optional int64			paramlong3	= 9;	//扩展参数
   optional string		paramstr1		= 10;	//扩展参数
   optional string		paramstr2		= 11;	//扩展参数
   optional string		paramstr3		= 12;	//扩展参数
} 